### PR TITLE
Section to Add Certificates

### PR DESCRIPTION
**Pull request by Vyshnav001**

### CODE REVIEW

File: components/CertificateCard.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                left: 0,
+                right: 0,
+                top: '7rem',
+                marginLeft: 'auto',
+                marginRight: 'auto',
+                width: '28rem',
+                height: '18rem',
+                borderRadius: '5%',
+              }}
+            //   className="shadow mb-3"
```
Comment (1hanzla100) [inline components/CertificateCard.tsx:38]: Remove Extra comments. and add some description in certification card

File: containers/Certificate.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { certificates } from '../portfolio';
+import { Container, Row } from 'reactstrap';
+import CertificateCard from '../components/CertificateCard';
+
+const Experience = () => {
```
Comment (1hanzla100) [inline containers/Certificate.tsx:6]: Rename this component to `Certificate`

**Review by 1hanzla100 — CHANGES_REQUESTED** [review]

**Comment by Vyshnav001** [comment]
- Extra comments are Removed
- Added description for Certificates 
- Component Renamed to Certificate

**Comment by 1hanzla100** [comment]
@Vyshnav001 Add Some space above title following other components.

**Comment by Vyshnav001** [comment]
> @Vyshnav001 Add Some space above title following other components.

Screenshot 2023-11-22 164206
 Is this ok?

**Pull request closed without merging**